context.go

```diff
@@ -51,7 +64,12 @@ func NewContext(handle *Handle) *Context {
 		return nil
 	}
 
-	return &Context{handle: handle, cContext: cContext}
+	timer, err := timer.NewTreeTimer(timer.WithBudget(budget), timer.WithComponents("_dd.appsec.waf.run"))
+	if err != nil {
```

Comment:
It kinda bugs me that we're dropping errors like this...
But since we don't really log things in this repo the alternative would be to change the API to return the error, meaning a new major V.
I guess we can put a pin in that for now but I think it's worth keeping in mind for when we plan a new major release. WDYT?

Reply:
We are starting to get a good list of things to do for the v3. I just added a task in jira to track [that](https://datadoghq.atlassian.net/browse/APPSEC-52238?atlOrigin=eyJpIjoiMGI0NThkOWFiODJhNDVmYzg1MmI5N2FlYjliYTQ5NjMiLCJwIjoiaiJ9)